helpers/autoCompleteHelper.js

```diff
@@ -29,11 +29,11 @@ async function addNotifyAutoCompleteOptions (interaction) {
 
     let filter
     if (subcommand === 'sub') {
-      filter = (name, subscribers) => (!optionValue || name.includes(optionValue)) && !subscribers.includes(userId)
+      filter = (name, subscribers) => (!optionValue || name.toLowerCase().includes(optionValue.toLowerCase())) && !subscribers.includes(userId)
```

Comment:
Would there be a way to sort the autocomplete as well?

Reply:
It is, there is a sort in the final statement
The comment about case insensitive sorting still holds though